src/cascadia/TerminalControl/TermControl.h

```diff
@@ -77,7 +77,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         void AdjustFontSize(int fontSizeDelta);
         void ResetFontSize();
 
-        winrt::fire_and_forget SwapChainChanged();
+        winrt::fire_and_forget RenderEngineSwapChainChanged();

```

Comment:
The nit-iest nit: Are these supposed to be public? Private? I'd think both private, and both with `_` prefixes on the names

Reply:
yeah i'm not sure why we made that one public

this file is huge and could use some organization